pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -82,46 +154,48 @@ task BuildStarSingleNucleus {
   }
 }
 
-
-
-workflow BuildIndices {
+task BuildBWAreference {
   input {
+    File genome_fa
+    File? chrom_sizes_file
+
+    # GTF annotation version refers to the version (GENCODE) or release (NCBI) listed in the GTF
+    String gtf_annotation_version
     # Genome source can be NCBI or GENCODE
     String genome_source
-    # GTF annotation version refers to the version or release listed in the GTF
-    String gtf_annotation_version
     # Genome build is the assembly accession (NCBI) or version (GENCODE)
     String genome_build
-    File annotations_gtf
-    File genome_fa
-    String gtf_annotation_version
-    File biotypes
+    # Organism can be Macaque, Mouse, Human, etc.
+    String organism
   }
 
-  # version of this pipeline
-  String pipeline_version = "2.1.0"
+  String reference_name = "bwa0.7.17-~{organism}-~{genome_source}-build-~{genome_build}-~{gtf_annotation_version}"
```

Comment:
I suggested removing the annotation from the file name (and ultimately the scATAC pipelineoutput) since we are not using a reference annotation to build this index. Not sur e if that makes sense or not.